content/en/contribute/medic/onboarding/daily-updates.md

```diff
@@ -0,0 +1,99 @@
+
+### Q: What is the best time to post updates on the daily channel?
+A: Whatever works best for you. But, usually, posting the update could be either one of the first activities you complete or one of the last things you do before leaving for the day. Here’s some interesting [reading](https://blog.xmtp.com/goodbye-standups-hello-wrap-ups/) on the topic.
+
+### Q: How is this measured in my performance review?
+A: Managers will share regular feedback throughout the year about the quality and consistency of your updates. Preferably, by the time performance reviews happen, you will already have received plenty of feedback and identified areas for growth that you committed to working on. Effective communication and reliability are part of our core competencies, and accountability is a big deal we are trying to get better at as a team, so status update consistency will be part of the calibration toward those competencies.
+
+### Q: How detailed should my updates be?
+A: We shared some [examples of updates]({{< relref "#bad-update-vs-good-update-examples" >}}) and some prompts that can help you draft your daily updates. Keep in mind that status updates demand clarity and specific information that is valuable and informs others. 
+
+### Q: What should I do if I have no significant updates for the day?
+A: If you don’t have significant updates about what you did, you can always share updates about how you feel about it. Maybe a task was simply straightforward and it doesn’t require many updates. Or maybe there were a number of difficult, unforeseen issues that came up, and you didn’t progress as much as you wanted. Maybe you had a super productive day. Tell the team how you feel; the channel is a safe space to speak honestly! (*Pro tip*: [Feelings Wheel](https://feelingswheel.com/))
+
+### Q: Is it okay to post challenges or roadblocks in the #dailies?
+A: Absolutely, someone can help you after reading your update. If you think of someone in particular who could give you a hand, just tag them!
+
+### Q: Should I respond to other team members' updates in the daily channel?
+A: If they tag you or you want to reply to something specific about their updates, it is highly encouraged to do so! You can also react to their messages with an emoji if you want to.
+
+### Q: What sort of feedback should I expect on my updates?
+A: Managers should provide direct, regular, both positive and constructive and helpful feedback on status updates provided by the team members.
+
+### Q: What if I'm alone in the product social time? 
+A: You don’t need to wait for more than 5 or 10 minutes if no one else shows up.
+
+### Q: How frequently should I go to the product social time?
+A: You are not required to join this call, but it is highly encouraged as we’d like teammates to connect not only via Slack or GitHub.
```

Comment:
Please move these 2 questions to the team-meetings page.

Reply:
Oh, my bad, thanks for flagging!